src/autoscaler/Makefile

```
@@ -8,11 +8,15 @@ PACKAGE_DIRS := $(shell $(GO) list ./... | grep -v /vendor/ | grep -v e2e)
 CGO_ENABLED = 0
 BUILDTAGS :=
 
+build-custom-metrics-cred-helper-plugin:
```

hmm why not keep the same build pattern ?
Suppose this will be in another repo eventually so its not important I guess

---

```suggestion
build: build-scalingengine build-metricsforwarder build-eventgenerator build-api build-metricsgateway build-metricsserver build-operator build-cred_plugin
```
and place the main.go into cred_plugin/cmd/cred_plugin/main.go